network/network.go

```diff
@@ -131,22 +135,30 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 // Collect implements the Collect method of the Collector interface.
 func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	hosts := []string{}
+	atomic.AddUint64(&c.scrapeID, 1)
+
+	scrapingStart := time.Now()
+	c.logger.Log("level", "info", "message", "collecting metrics", "scrapeID", c.scrapeID)
 
 	service, err := c.kubernetesClient.CoreV1().Services(c.namespace).Get(c.service, metav1.GetOptions{})
 	if err != nil {
-		c.logger.Log("level", "error", "message", "could not get service", "stack", microerror.Stack(err))
+		c.logger.Log("level", "error", "message", "could not get service from kubernetes api ", "scrapeID", c.scrapeID, "stack", microerror.Stack(err))
 		c.errorCount.Inc()
 		return
 	}
 
+	c.logger.Log("level", "info", "message", "collected service", "service ", c.service, "scrapeID", c.scrapeID)
 	hosts = append(hosts, fmt.Sprintf("%v:%v", service.Spec.ClusterIP, c.port))
 
+	c.logger.Log("level", "info", "message", "connecting to kubernetes api to get endpoints", "service", c.service, "scrapeID", c.scrapeID)
```

Comment:
```suggestion
	c.logger.Log("level", "info", "message", "collecting endpoints", "endpoint", c.service, "scrapeID", c.scrapeID)
```

Reply:
Why is it better to set endpoint key to service value?

Reply:
We are getting an endpoint here, it happens that the endpoint has the same name as the service here. But it feels weird to me to read a message about an endpoint which give the name of a service.